app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC
+from ..player import Player

```

Comment:
Use absolute imports instead of relative ones to not be dependent on the project structure. Apply this change to the other places as well.